File: examples/sample.dart
import 'dart:io';

class Calculator {
  int add(int a, int b) {
    return a + b;
  }

  double multiply(double x, double y) {
    return x * y;
  }
}

String formatMessage(String message) {
  return 'Message: $message';
}

void processList(List<int> numbers) {
  for (int num in numbers) {
    print(num);
  }
}

Future<void> asyncOperation() async {
  await Future.delayed(Duration(seconds: 1));
  print('Async operation completed');
}

File: examples/sample.java
import java.util.List;
import java.util.ArrayList;

public class Calculator {
    public int add(int a, int b) {
        return a + b;
    }
    
    public double multiply(double x, double y) {
        return x * y;
    }
    
    private String formatMessage(String message) {
        return "Message: " + message;
    }
}

class StringProcessor {
    public static void processList(List<String> items) {
        for (String item : items) {
            System.out.println(item);
        }
    }
    
    protected boolean isValid(String input) {
        return input != null && !input.trim().isEmpty();
    }
} 